clicache/integration-test2/GarbageCollectCache.cs

```diff
@@ -83,10 +83,9 @@ public void VerifyNoLeakedThreads()
                     // environment startup.
                     if (i > 5)
                     {
-                        //Assert.True(.8 < ratio && ratio < 1.3);
                         string error = "ncThreadsBefore = " + ncThreadsBefore.ToString() +
                             ", ncThreadsAfter = " + ncThreadsAfter.ToString();
-                        Assert.False(!(.8 < ratio && ratio < 1.3), error);
+                        Assert.False(!(.5 < ratio && ratio < 2.0), error);
```

Comment:
Can we please make this a positive assert, rather than a negative?  It's very hard to read.  The commented-out line that you removed is MUCH clearer, while stating the same thing.  
```
Assert.True(.5 < ratio < 2.0);
```

Even changing it to two different assertions is easier to read, and potentially more helpful for debugging/diagnosis:

```
Assert.False(ratio < .5, error);
Assert.False(ratio > 2.0, error);
```

Reply:
I put that in for debugging purposes. Assert.True wasn't giving me the output. Agree with your comment. Will change it back.